src/plugins/intel_cpu/src/transformations/transformation_pipeline.cpp

```diff
@@ -849,6 +851,9 @@ void Transformations::PostLpt() {
     }
     CPU_REGISTER_PASS_COMMON(postLPTPassManager, ov::pass::transpose_sinking::TSShapeOfForward);
     CPU_REGISTER_PASS_COMMON(postLPTPassManager, StatefulSDPAFusion);
+    // markup Rope Input only when BF16 inference.
+    if (inferencePrecision == ov::element::bf16)
```

Comment:
What about FP16 precison? Shouldn't we force fp32 as well?

Reply:
I don't see accuracy issue reported by FP16, but I can add this as well